web/src/views/record_my_sleep/index.js

```diff
@@ -0,0 +1,250 @@
+import React from 'react';
+import { Card, CardBody, Container, Row, Col } from 'reactstrap';
+import { Alarm, Hotel, Timeline } from '@material-ui/icons';
+
+import Header from 'components/header';
+import Table from 'components/table';
+
+import text from './text.json';
+
+import WIPWarning from 'components/wip_warning';
+
+import material_needed from './material_needed.json';
+import AlternatingTextImage from 'components/alternating_text_image';
+
+import ElectrodesDisplay from './electrodes_display';
+
+const RecordMySleep = () => {
+  return (
+    <>
+      <Header
+        sizeClass={'pb-150'}
+        shapeQty={7}
+        title={text['header_title']}
+        subtitle={text['header_subtitle']}
+        description={text['header_description']}
+      />
+      <section className="section">
+        <Container className="text-justify pt-lg">
+          <h3 className="display-3">Create your own sleep lab</h3>
+          <p className="lead">
+            Here are some explanations about how to get your own EEG data in order to score your sleep using this
+            website. If you already own an OpenBCI board, you could have very little (see nothing) to buy.
+          </p>
+          <p className="lead">
+            If you don’t have the time or the resources to create your own sleep lab, we have pre-recorded one of our
+            members' night's sleep. You can take a look at the end result right here:
+          </p>
+        </Container>
+      </section>
+
+      <section className="section bg-secondary">
+        <Container className="text-justify pt-lg">
+          <h3 className="display-3">What you need</h3>
```

Comment:
Un peu de margin bottom et une petite intro à la liste d'éléments pourrait rendre ça un peu moins dry :) dans la maquette on avait mis ça: `If you already own an OpenBCI board, you could have very little to buy. Either ways, you are going to need the following:`
```suggestion
          <h3 className="display-3 mb-3">What you need</h3>
```